src/reference-manual/types.Rmd

```diff
@@ -1327,6 +1327,54 @@ void pretty_print_tri_lower(matrix x) { ... }
 
 ```
```

Comment:
I like the new table!

Line 1327 shouldn't be blank.

This new table needs a description/intro.

Is there a way to get `(unsized)`, etc. up with the header?  If not, no big deal---it's just awkward having them with the content.

Also, could you add an example for an array of vectors?

Reply:
Array of vectors added with a note that arrays in general follow a similar pattern and moved up the subheaders (required changing to 'grid style' markdown table).

I agree a description is needed. Some of this is related to https://github.com/stan-dev/docs/pull/394#discussion_r704440216 and how we want to handle the de-duplication of this information